site/layouts/shortcodes/pluginjsdocs.html

```diff
@@ -29,7 +29,7 @@ <h5 id="{{ .Name }}">{{ .Name }}</h3>
 {{ define "JSFunctionSignatureHTML" }}
     <pre><code>/**
 {{- range .Comments }}
-    * {{ . }}
+    *{{ . }}
```

Comment:
I've removed the `trim` call in the comments strings, so there were two spaces here. I think it is better to rely on the formatting of the comments, as they sometimes use it in order to make the text clearer. See, for example:
![image](https://user-images.githubusercontent.com/3924815/82805035-996a3400-9e83-11ea-85f0-0d0ef1fddba9.png)